src/com/nutrons/framework/subsystems/Settings.java

```diff
@@ -1,15 +1,22 @@
 package com.nutrons.framework.subsystems;
 
 import com.nutrons.framework.Subsystem;
+import com.nutrons.framework.util.FlowOperators;
+import edu.wpi.first.wpilibj.Preferences;
+import io.reactivex.Flowable;
 
-public interface Settings extends Subsystem {
-  /**
-   * Get int from settings table.
-   */
-  int getInt(String key);
-
-  /**
-   * Get double from settings table.
-   */
-  double getDouble(String key);
-}
+public class Settings implements Subsystem {
+
+    public Flowable<Double> getProperty(String key) {
+        return FlowOperators.toFlow( () ->Preferences
+                .getInstance()
+                .getDouble(key, 0.0) )
+                .distinctUntilChanged();
+        //TODO: don't emit anything if it's 0.0, store last value and send that as backup
```

Comment:
I'm not sure how to do this yet - I'll leave this as a TODO until I consult a mentor/Asher